packages/lumx-react/src/components/switch/Switch.tsx

```diff
@@ -44,7 +44,7 @@ interface SwitchProps extends GenericProps {
     useCustomColors?: boolean;
 
```

Comment:
Should we harmonize the `checked`/`value` props? The `CheckBox` uses `value`, the Switch uses `checked` and the `RadioButton` uses `checked`

Aesthetically I would prefer `value` but I think we probably should use `checked` because it's the HTML standard. The `value` is also used in these elements to have a string representation of the boolean value.
It is especially useful if you name a group of `RadioButton` each having a string value. You basically get a field with enumerated string value rather than multiple boolean values.

Although to make this truly work we probably should change our `onChange` type to `onChange(value: V, name: string, event: SyntheticEvent): void` or `onChange(value: V, name: string, checked: boolean, event: SyntheticEvent): void`?
I'm really not sure on this one, maybe we could debate